docs/source/how_tos/visualize_data_with_tableau.rst

```diff
@@ -0,0 +1,46 @@
+***************************
+Visualize Data with Tableau
+***************************
+`Tableau <https://www.tableau.com/>`_ is a nice commercial tool for exploring and visualizing
+tabular data. Licenses are available to NRELians.
+
+In addition to making visualizations, Tableau makes it easy to select, filter, group, and describe
+your data in tables. This can be easier than the same operations in a Python REPL with ``pyspark``
+or ``pandas``.
+
+This page describes various ways to connect Tableau to dsgrid data after you've installed Tableau
+Desktop on your local computer.
+
+Parquet files on a local computer
+=================================
+This can be accomplished by connecting Tableau to DuckDB.
+
+1. Copy the Parquet files to your computer.
+
+2. Install `DuckDB <https://duckdb.org/docs/installation/>`_. Next to ``Environment``, select
+   ``Command line``.
+
+3. Install a JDBC driver and connect Tableau to DuckDB by following `DuckDB's
+   <https://duckdb.org/docs/guides/data_viewers/tableau>`_ instructions.
+
+   The documentation provides instructions for a "taco" connector. This does not appear to be
+   necessary for basic operations. You will likely benefit from it if you perform advanced SQL
+   operations.
```

Comment:
Is it clear in DuckDB's instructions how to do something similar if that's what the user wants? I.e., what instructions should the user follow if they only want "basic operations"?

Reply:
I'll just remove the paragraph and leave it to the user on whether they want to follow DuckDB's advice.